sklearn/kernel_approximation.py

```diff
@@ -519,3 +652,16 @@ def _get_kernel_params(self):
             params['coef0'] = self.coef0
 
         return params
+
+# Relative entropy for tests
```

Comment:
Because amueller told me to implement it either in kernel_approximation or in utils.fixes (see above comments).